aws/v12.0.0/release.yaml

```
@@ -0,0 +1,60 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v12.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - componentVersion: 0.9.0
+    name: cert-manager
+    version: 1.0.8
+  - name: chart-operator
+    version: 0.13.2
+  - componentVersion: 1.17.3
+    name: cluster-autoscaler
+    version: 1.17.3
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 0.7.2
+    name: external-dns
+    version: 1.2.2
+  - componentVersion: 3.6.0
+    name: kiam
+    version: 1.3.1
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.1.1
+  - componentVersion: 0.3.6
+    name: metrics-server
+    version: 1.1.1
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.3.0
+  components:
+  - name: app-operator
+    version: 1.1.9
```

```suggestion
    version: 1.0.0
```

Sadly the versioning here is weird.

For Helm 2 we need the version to be 1.0.0. This is processed by app-operator 1.09 in the app collection. e.g. https://github.com/giantswarm/aws-app-collection/blob/0ab639b019a93e6b9fc8548b92b9980ac6b1daaf/helm/aws-app-collection-chart/templates/app-operator-unique-1.0.9.yaml

This is one of the things we will fix with Helm 3 for tenant clusters. The version will be 2.X and there will be an app-operator instance per version like cluster-operator.

cc @njuettner